### PR TITLE
[7] Develop a Contact Form Page Using HTML

### PR DESCRIPTION
**Pull request by Benedictcares**
## Task: `Develop a Contact Form Page Using HTML`
- `added Contact Page `

### CODE REVIEW

File: submissions/ejikeIweriebor/html/Contact.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html
+<html lang=ENG>
+    <meta charset UTF-8>
+    <head>
+        <title>CONTACT FORM</title>
+    </head>
+    <body>
+        <h1>SPENDTHRIFT!</h1>
+        <form action="#" method="post" enctype="application/x-www-form-urlencoded">
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/Contact.html:9]: Remove attributes `action`, `method` and `enctype` they are not needed now.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Benedictcares update the name of your file to be in all lowercase.

**Review by Benedictcares — COMMENTED** [review]
I have done that, thank you

File: submissions/ejikeIweriebor/html/contact.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                <label name="firstname">First Name</label>
+                <input type="text" id="firstname" name="firstname"/><br><br>
+                <label for="lastname">Last Name</label>
+                <input type="text" id="lastname" name="lastname"/><br><br>
+                <label for="email">Email</label>
+                <input type="email" id="email" name="email" placeholder="Enter a valid email"/><br>
+               
+                <label  for="subject">Subject</label>
+
+                <select name="Subject" id="subject">
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/contact.html:21]: @Benedictcares select `name` should be in lowercase

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**